front/app/containers/Admin/pages-menu/containers/HeroBannerForm/index.tsx

```diff
@@ -0,0 +1,492 @@
+                    banner_signed_out_header_overlay_opacity ||
+                    theme.signedOutHeaderOverlayOpacity
+                  }
+                  onChange={debouncedHandleOverlayOpacityOnChange}
+                />
+              </SectionField>
+            </>
+          )}
+        <SectionField key={'banner_text'}>
+          <SubSectionTitle>
+            <FormattedMessage {...messages.bannerTextTitle} />
+          </SubSectionTitle>
+          <InputMultilocWithLocaleSwitcher
+            type="text"
+            valueMultiloc={banner_signed_out_header_multiloc}
+            label={
+              <Box display="flex" mr="20px">
+                <FormattedMessage {...messages.bannerHeaderSignedOut} />
+              </Box>
+            }
+            maxCharCount={TITLE_MAX_CHAR_COUNT}
+            onChange={handleSignedOutHeaderOnChange}
+            errorMultiloc={headerAndSubheaderErrors.signedOutHeaderErrors}
+          />
+        </SectionField>
+        <SectionField>
+          <InputMultilocWithLocaleSwitcher
+            type="text"
+            valueMultiloc={banner_signed_out_subheader_multiloc}
+            label={formatMessage(messages.bannerHeaderSignedOutSubtitle)}
+            maxCharCount={SUBTITLE_MAX_CHAR_COUNT}
+            onChange={handleSignedOutSubheaderOnChange}
+            errorMultiloc={headerAndSubheaderErrors.signedOutSubheaderErrors}
+          />
+        </SectionField>
+        <SectionField>
+          <InputMultilocWithLocaleSwitcher
+            type="text"
+            valueMultiloc={banner_signed_in_header_multiloc}
+            label={formatMessage(messages.bannerHeaderSignedIn)}
+            onChange={handleSignedInHeaderOnChange}
+          />
+        </SectionField>
+        <SectionField key="avatars">
+          <SubSectionTitle>
+            <FormattedMessage {...messages.avatarsTitle} />
+          </SubSectionTitle>
+          <Setting>
+            <ToggleLabel>
+              <StyledToggle
+                checked={!!banner_avatars_enabled}
+                onChange={() => {
+                  updateValueInLocalHomepageSettings(
+                    'banner_avatars_enabled',
+                    !banner_avatars_enabled
+                  );
+                }}
+              />
+              <LabelContent>
+                <LabelTitle>
+                  {formatMessage(messages.bannerDisplayHeaderAvatars)}
+                </LabelTitle>
+                <LabelDescription>
+                  {formatMessage(messages.bannerDisplayHeaderAvatarsSubtitle)}
+                </LabelDescription>
+              </LabelContent>
+            </ToggleLabel>
+          </Setting>
+        </SectionField>
+        <Outlet
+          id="app.containers.Admin.settings.customize.headerSectionEnd"
+          homepageSettings={localHomepageSettings}
+          handleOnChange={handleSettingOnChange}
+          // testing
+          errors={{ base: [{ error: 'some error' }] }}
```

Comment:
This is WIP and related to the validation issues with the custom button Multiloc (see `CustomizedButtonSettings`)